src/20241127_Multi_FluidAlignment/AaveV3Ethereum_FluidAlignment_TokenTransfers_20241127.t.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {GovV3Helpers} from 'aave-helpers/src/GovV3Helpers.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+
+import 'forge-std/Test.sol';
+import {ProtocolV3TestBase, ReserveConfig} from 'aave-helpers/src/ProtocolV3TestBase.sol';
+import {AaveV3Ethereum_FluidAlignment_TokenTransfers_20241127} from './AaveV3Ethereum_FluidAlignment_TokenTransfers_20241127.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+
+/**
+ * @dev Test for AaveV3Ethereum_FluidAlignment_TokenTransfers_20241127
+ * command: FOUNDRY_PROFILE=mainnet forge test --match-path=src/20241127_Multi_FluidAlignment/AaveV3Ethereum_FluidAlignment_20241127.t.sol -vv
+ */
+contract AaveV3Ethereum_FluidAlignment_TokenTransfers_20241127_Test is ProtocolV3TestBase {
+  AaveV3Ethereum_FluidAlignment_TokenTransfers_20241127 internal proposal;
+
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 21280537);
+    proposal = new AaveV3Ethereum_FluidAlignment_TokenTransfers_20241127();
```

Comment:
```suggestion
    proposal = new AaveV3Ethereum_FluidAlignment_TokenTransfers_20241127();
    vm.startPrank(proposal.INSTADAPP_TREASURY());
    IERC20(proposal.INST_TOKEN()).approve(AaveV3Ethereum.ACL_ADMIN, proposal.INST_AMOUNT());
    vm.stopPrank();
```

Reply:
the ci must pass for ipfs to be uploaded - if allowance is missing, you need to at least mock it.